scss/cayman.scss

```diff
@@ -2,19 +2,31 @@
 $large-breakpoint: 64em;
 $medium-breakpoint: 42em;
 
-// Sass variables
-$brand-color: #159957;
-$secondary-brand-color: #155799;
-$light-color: #FFFFFF;
+// Headers
+$header-heading-color: #FFFFFF;
+$header-bg-color: #159957;
+$header-bg-color-secondary: #155799;
+$headings-color: #159957;
```

Comment:
Having `$headings-color` grouped with the page header stuff is kind of confusing. How about we rename this to `$section-headings-color` and move it down with `$body-text-color` etc.